test/unit/calculators/state_pension_age_calculator_test.rb

```diff
@@ -343,5 +343,25 @@ class StatePensionAgeCalculatorTest < ActiveSupport::TestCase
         end
       end
     end
+  
+    context "#rolling_increase_period?" do
+      should "be true for a date that is between 06/04/1970 and 05/04/1978" do
+        @calculator = StatePensionAgeCalculator.new(
+          gender: "male", dob: Date.parse("6 April 1970"))
```

Comment:
Does gender affect this calculation? It looks a bit misleading if it doesn't affect the outcome, so maybe we could we skip it? (Unless that causes an error.)